app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,24 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Betsy</title>
+  <title>Gems on Gems on Gems</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <link href='https://fonts.googleapis.com/css?family=IM+Fell+DW+Pica' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Tinos:400,400italic' rel='stylesheet' type='text/css'>
+  <link href='https://fonts.googleapis.com/css?family=Sacramento' rel='stylesheet' type='text/css'>
```

Comment:
When using Google fonts you can add several fonts to a collection and use one reference. Also, It seems like one or more of the fonts are being used. If that's the case, remove them. Loading fonts can take up time.